locales/locales/en.json

```diff
@@ -189,6 +189,49 @@
         "message": "simple arrow"
     },
 
+    "accountSwitcherCliHelp": {
+      "message": "switch users to [username]"
+    },
+
+    "accountSwitcherCliSwitchToUsernamePrompt": {
+      "message": "Switch to username:"
+    },
+
+    "accountSwitcherCliUserinfoError": {
+      "message": "Error loading userinfo for"
+    },
+
+    "accountSwitcherGoldUntil": {
+      "message": "Until"
+    },
+
+    "accountSwitcherAccountSwitchError": {
+      "message": "Could not switch accounts. Reddit may be under heavy load. Please try again in a few moments."
+    },
+
+    "accountSwitcherLoginError": {
+      "message": "Could not log in as <b>$1</b> because either the username or password is wrong.
```

Comment:
We want to avoid HTML in translations because it's awkward to translate (and because it's unsafe). In this case, since these are rare error messages, we can go without the extra formatting and just use `\n`.

Also, JSON strings can't be multiline, so this (and below) is invalid syntax.